src/sentry/integrations/pagerduty/integration.py

```diff
@@ -164,6 +164,6 @@ def dispatch(self, request, pipeline):
             pipeline.bind_state("config", request.GET["config"])
             return pipeline.next_step()
 
-        account_name = getattr(request, "account", None)
+        account_name = request.GET.get("account", None)
```

Comment:
@MeredithAnya Just curious, why didn't unit tests catch this bug?

Reply:
looks like I only tested the install flow without that parameter and updating the services manually. this would actually be a good test to add so I can do that

Reply:
@MeredithAnya Yes, I would suggest adding a test for this case. Thanks!